Content.Shared/Weapons/Ranged/Components/GunComponent.cs

```diff
@@ -229,6 +229,12 @@ public sealed partial class GunComponent : Component
     /// </summary>
     [DataField]
     public bool ClumsyProof = false;
+
+    /// <summary>
+    ///     The percentage chance of a given gun to accidentally discharge if violently thrown into a wall or person
+    /// </summary>
+    [DataField]
+    public float Reliability = 0.1f;
```

Comment:
So higher reliability the worse?

Reply:
Correct. :trollface:

Reply:
Should have a more clear name.

Reply:
I changed the name to FireOnDropChance, so that its much clearer what it does.